examples/Rules-method-NextBestNCRMLoss.R

```diff
@@ -0,0 +1,63 @@
+# nolint start
```

Comment:
hm ok, otherwise just use snake_case for variable names below, then lintr should be ok